packages/hdwallet-tally/src/tally.ts

```diff
@@ -0,0 +1,273 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import * as eth from "./ethereum";
+import _ from "lodash";
+
+export function isTally(wallet: core.HDWallet): wallet is TallyHDWallet {
+  return _.isObject(wallet) && (wallet as any)._isTally;
+}
+
+
+export class TallyHDWallet implements core.HDWallet, core.ETHWallet {
+  readonly _supportsETH = true;
+  readonly _supportsETHInfo = true;
+  readonly _isTally = true;
+
+
+  info: TallyHDWalletInfo & core.HDWalletInfo;
+  ethAddress?: string | null;
+  provider: any;
+
+  constructor(provider: unknown) {
+    this.info = new TallyHDWalletInfo();
+    this.provider = provider
+  }
+
+  async getFeatures(): Promise<Record<string, any>> {
+    return {};
+  }
+
+  public async isLocked(): Promise<boolean> {
+    return !this.provider.tally.isUnlocked();
+  }
+
+  public getVendor(): string {
+    return "Tally";
+  }
+
+  public async getModel(): Promise<string> {
+    return "Tally";
+  }
+
+  public async getLabel(): Promise<string> {
+    return "Tally";
+  }
+
+  public async initialize(): Promise<void> { 
+    // nothing to initialize
+  }
+
+  public hasOnDevicePinEntry(): boolean {
+    return this.info.hasOnDevicePinEntry();
+  }
+
+  public hasOnDevicePassphrase(): boolean {
+    return this.info.hasOnDevicePassphrase();
+  }
+
+  public hasOnDeviceDisplay(): boolean {
+    return this.info.hasOnDeviceDisplay();
+  }
+
+  public hasOnDeviceRecovery(): boolean {
+    return this.info.hasOnDeviceRecovery();
+  }
+
+  public hasNativeShapeShift(srcCoin: core.Coin, dstCoin: core.Coin): boolean {
+    return this.info.hasNativeShapeShift(srcCoin, dstCoin);
+  }
+
+  public supportsOfflineSigning(): boolean {
+    return false;
```

Comment:
```suggestion
    // Keep an eye on the status of the refactor PR here: https://github.com/tallycash/extension/pull/1165/files. This will add offline signing support to Tally, at which point this should return true.
    return false;
```